aws-auth-cognito/src/test/java/featureTest/utilities/CognitoMockFactory.kt

```diff
@@ -67,6 +68,7 @@ class CognitoMockFactory(
                     setupError(mockResponse, responseObject)
                     SignUpResponse.invoke {
                         this.codeDeliveryDetails = parseCodeDeliveryDetails(responseObject)
+                        this.userConfirmed = if(responseObject.containsKey("userConfirmed")){(responseObject["userConfirmed"] as JsonPrimitive).boolean} else false
```

Comment:
use "as?" and for readability split this into more than 1 line since we have if and else.

Reply:
➕

Reply:
Done